record-and-playback/core/lib/recordandplayback/edl/media_utils.rb

```
@@ -0,0 +1,102 @@
+            pid = spawn(
+              *ffprobe_cmd,
+              close_others: true,
+              in: :close,
+              out: w,
+              err: log_file,
+            )
+            w.close
+
+            prev_end = 0
+
+            csv = CSV.new(r, headers: %i[pts_time duration_time], converters: %i[float float], skip_blanks: true, empty_value: 'N/A')
```

RuboCop (1.85.1)

[convention] Line is too long. [133/128]

(Layout/LineLength)